docs/designs/output.md

```diff
@@ -114,80 +114,14 @@ Different files can share the same `{site-url}` or `{site-path}` due to versioni
-### Files
-
-To efficiently look up a file from URL for dynamic rendering, 
-`docfx` produces a manifest file at `_site/build.manifest` that lists all the output files and properties needed for lookup.
-
-*locale*, *moniker* are special properties that flows dynamically when users navigate from page to page.
-These are called *browser navigation properties* and are stored in manifest.
-
-```javascript
-{
-    "files": [
-        {
-            "siteUrl": "/dotnet/api/system.string",
-            "outputPath": "en-us/netstandard-2.0/dotnet/api/system.string.json",
-            "locale": "en-us",
-            "moniker": "netstandard-2.0", 
-
-            // other browser navigation properties...
-            // other properties needed for backward compatibility
-        }, 
-        {
-            "siteUrl": "/dotnet/api/system.string",
-
-            // when `outputPath` does not exist,
-            // use `sourcePath` relative to source docset folder to locate the file
-            "sourcePath": "en-us/netstandard-2.0/dotnet/api/system.string.json"
-        }
-    ]
-}
-```
+## System Generated Outputs
 
-We only save *browser navigation properties* for images in `build.manifest`. Other image metadata are discarded.
-
-Different verions of the same document, or difference locales of the same document may have the same output content.
-It is nessesary to duplicate them for static rendering. To save space for dynamic rendering, set the `path`
-property to the same location.
-
-### Dependencies
-
-To efficiently look up the impacted files for a given file,
-`docfx` produces a dependency list at `_site/build.manifest` that lists all the relationships between different files.
-
-*Dependency impacts* means the impacts(build/publish) to the referenced file causing by one or more file changes:
-
-  - **Content impact** means the changing file content has impact to the referenced file(s), like token/codesnippet, overwrite markdown.
-    - Inclusion(token/codesnippet) reference, like `![include](token file path)`.
-    - Overwrite markdown reference(TODO).
-  - **URL impact** means the changing file output URL has impact to the referenced file(s), like file link, UID reference.
-    - File link reference, like `[link title](file path)`.
-    - UID reference, like `<xref:System.Char>`.
-    - Metadata reference, like `toc_rel: ../TOC.json`.
-
-```json
-{  
-   "dependencies":[  
-      {  
-         "dotnet/azure/service-less-app.md": [
-            {  
-               "source":"dotnet/azure/service-less-app-dependent.md",
-               "type":"link"
-            },
-            {  
-               "source":"dotnet/azure/service-less-app-token.md",
-               "type":"inclusion"
-            }
-         ]
-      }
-   ]
-}
-```
+Besides content files, `docfx` generates miscellaneous files during the build. The name of these system generated fiels starts with `.`. If the format is JSON, then the extension MUST be `.json` for faster DHS download speed.
```

Comment:
it's better to not mention our low lever consumer in our design document, we don't want to be limited by them